### PR TITLE
Created Profile Details Page

### PR DESCRIPTION
**Pull request by rajivtitus**
### What this PR does (required):
- Created profile details page
- Added app title, logo and meta description
- Used mock profile object to render data

Note: font is different, I have not changed the font as I have already done so in a previous branch.

### Screenshots / Videos (required):
Capture

### Any information needed to test this feature (required):
- `npm start` on client | `npm run dev` on server
- Enter `/profile/<any id>`

At the moment our app does not create a profile in the BE, this branch only replicates the UI for Profile Details page.
Once our BE model is finalized, we will setup functionality for creating as well as fetching profiles.

To test the app, as it is an unprotected route, before login or signup (if you are logged in, please logout) and enter below in the url:

http://localhost:3000/profile/<random id>

eg: http://localhost:3000/profile/1234

### Any issues with the current functionality (optional):
- None

### CODE REVIEW

**Comment by dproc96** [comment]
Could you include in the testing instructions the process for creating a profile, it looks like one isn't created when you sign up

**Comment by rajivtitus** [comment]
Hi @dproc96 , 
At the moment our app does not create a profile in the BE, this branch only replicates the UI for Profile Details page.
Once our BE model is finalized, we will setup functionality for creating as well as fetching profiles.

To test the app, as it is an unprotected route, before login or signup (if you are logged in, please logout) and enter below in the url:
`http://localhost:3000/profile/<random id>`

eg: `http://localhost:3000/profile/1234`

File: client/src/components/ProfileDetails/About/About.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+          </Typography>
+          <Typography variant="body1" className={classes.aboutDescription}>
+            {profile.description}
+          </Typography>
+        </Box>
+        <GridList cellHeight={115} cols={5}>
+          {profile.gallery.map((photo, index) => (
+            <GridListTile cols={1} className={classes.galleryTile} key={index}>
+              <img src={photo} alt="Profile" />
+            </GridListTile>
```
Comment (sina-jamshidi) [inline client/src/components/ProfileDetails/About/About.tsx:52]: 👍

File: client/src/pages/ProfileDetails/ProfileDetails.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  gallery: [
+    'https://upload.wikimedia.org/wikipedia/commons/thumb/e/e5/Axel%2C_the_English_Bulldog.jpg/170px-Axel%2C_the_English_Bulldog.jpg',
+    'https://images.pexels.com/photos/3715587/pexels-photo-3715587.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500',
+  ],
+  city: 'Toronto, Ontario',
+  description:
+    'Animals are my passion! I will look after your pets with loving care. I have some availability for pet care in my home as well. I have 10 yrs experience at the Animal Hospital, and have owned multiple pets for many years, including numerous rescues. Kindly email, text or call me and I will respond promptly!',
+  price: 18,
+  rating: 4.35,
+};
```
Comment (sina-jamshidi) [inline client/src/pages/ProfileDetails/ProfileDetails.tsx:23]: great idea

**Review by sina-jamshidi — APPROVED** [review]
Solid PR! Looks good to go!

**Pull request merged**